### PR TITLE
feat(verify email): fixed typo with EMail to Email

### PR DESCRIPTION
**Pull request by milan-bc**
## Description (optional)
Add a concise explanation of the changes.

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/data/analytics/types.ts [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-import { MempoolVerifyEMailType } from 'data/components/types'
+import { MempoolVerifyEmailType } from 'data/components/types'
 import { RemoteDataType } from 'core/types'
 
 // types
 export type ABTestNameType = AB_TESTS
 
 export type ABTestCmdType = {
-  command: MempoolVerifyEMailType
+  command: MempoolVerifyEmailType
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/data/analytics/types.ts:10]: What does this have to do with the mempool?

Comment (milan-bc) [reply packages/blockchain-wallet-v4-frontend/src/data/analytics/types.ts:10]: Nothing, good point removing it 👍

**Review by schnogz — APPROVED** [review]
